src/04-modality-specific-files/09-positron-emission-tomography.md

```diff
@@ -219,6 +219,16 @@ We refer to the common principles for the standards for describing dates and tim
 
 All reconstruction-specific parameters that are not specified, but one wants to include, should go into the `ReconMethodParameterValues` field.
 
+#### Task related metadata
+
+| **Key name**    | **Requirement level** | **Data type** | **Description**                                                                                                                                                                                                                                                                                                                                                                      |
+| ----------------| --------------------- | ------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------ |
+| TaskName        | RECOMMENDED           | [string][]    | Name of the task. No two tasks should have the same name. The task label included in the file name is derived from this `TaskName` field by removing all non-alphanumeric (`[a-zA-Z0-9]`) characters. For example `TaskName` `"faces n-back"` will correspond to task label `facesnback`. A RECOMMENDED convention is to name resting state task using labels beginning with `rest`. |
```

Comment:
I would say that 99.9% of PET studies don't carry out a task (e.g. combined PET and fMRI). Therefore, this seems to be taking a lot of space, and may potentially confuse PET people. Wouldn't it be possible in the task section in the top to refer to this metadata in the behavioral experiments?

Reply:
Actually forgot to mention but at our last at our last maintainers meeting we said that we'll this out of PET for now: the long term plan would to refactor task related stuff into a separate sections that we can "point" to from all concerned modalities. 

I would then get added to PET at this time.

So I will remove it from PET in this PR for now.